support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -423,21 +423,21 @@ ApparentHorizons:
       DivergenceTol: 1.2
       DivergenceIter: 5
       MaxIts: 100
-    Verbosity: Quiet
+    Verbosity: Silent
```

Comment:
Leave these as quiet. We don't write this information out anywhere else (even though maybe we should) and it's useful to know. Also, it's not affecting performance (that I know of)